# Behaviour model
string model        # Base58 
# Task
string objective     # Base58
# Operational token
string token        # address
# Execution cost
uint32 cost         # int
# Settlement fee
uint32 lighthouseFee # int 
# Deadline block number
uint32 deadline
# Order nonce value 
uint8[] nonce       # hex
# Order signature
uint8[] signature   # hex
